README.md

```diff
@@ -34,8 +37,9 @@ var add = function (x, y) {
   return x + y
 }
 ```
-
--- YOUR ANSWER HERE --
+es5
+yes....? because variable add references the anonymous function but the fuction will be asocciated with variable add
```

Comment:
Yes it is. It's a function expression. Where an anonymous function is assigned to a variable. 💯